build.gradle

```diff
@@ -171,14 +170,14 @@ dependencies {
     testImplementation 'com.tngtech.java:junit-dataprovider:1.13.1'
     testImplementation 'net.javacrumbs.json-unit:json-unit-assertj:2.17.0'
 
-    testImplementation 'net.serenity-bdd:serenity-core:2.1.8'
-    testImplementation 'net.serenity-bdd:serenity-junit:2.1.8'
-    testImplementation 'net.serenity-bdd:serenity-spring:2.1.8'
+    testImplementation 'net.serenity-bdd:serenity-core:2.2.7'
```

Comment:
I tried upgrading this and it broke everything

Reply:
I even raised https://dsdmoj.atlassian.net/browse/DT-666 to fix

Reply:
I've reverted this.. would be nice to get rid entirely but obviously a ton of work. 
We can't move to junit 5 until it goes